Detector/DetSensitive/src/AggregateCalorimeterSD.cpp

```diff
@@ -58,9 +58,14 @@ bool AggregateCalorimeterSD::ProcessHits(G4Step* aStep, G4TouchableHistory*) {
   }
   // if not, create a new hit
   // deleted in ~G4Event
-  hitMatch = new dd4hep::sim::Geant4CalorimeterHit(pos);
+  hitMatch = new fcc::Geant4CaloHit(0, // track->GetTrackID()
+                                    0, // track->GetDefinition()->GetPDGEncoding()
+                                    edep,
+                                    0 // track ->GetGlobalTime()
+                                    ) ;
+
+  hit->position = midPos;
```

Comment:
Is this `hit` related to `hitMatch` at this point?

Reply:
good catch, this should be hitMatch